### PR TITLE
Translation amqp-dev-services

### PR DESCRIPTION
**Pull request by dellamas**
Hi everyone!

Translation : Dev Services for AMQP
Please check if everything is okay.

Note: This is my first contribution to an Open Source project. The first of many!

### CODE REVIEW

File: l10n/po/pt_BR/_guides/amqp-dev-services.adoc.po [hunk 1]
```diff
@@ -57,69 +55,61 @@ msgstr "o `amqp-host` ou `amqp-port` está configurado"
...
 msgid "Configuring the image"
-msgstr "Configurar a imagem"
+msgstr "Configurarando a imagem"
 
 #. type: Plain text
 #: upstream/_guides/amqp-dev-services.adoc:50
-#, fuzzy
 msgid "Dev Services for AMQP uses https://quay.io/repository/artemiscloud/activemq-artemis-broker[activemq-artemis-broker] images.  You can configure the image and version using the `quarkus.amqp.devservices.image-name` property:"
-msgstr "Os serviços de desenvolvimento para AMQP usam imagens  link:https://quay.io/repository/artemiscloud/activemq-artemis-broker[activemq-artemis-broker]. É possível configurar a imagem e a versão usando a propriedade `quarkus.amqp.devservices.image-name`:"
+msgstr "O Dev Services para AMQP usa a  imagem  link:https://quay.io/repository/artemiscloud/activemq-artemis-broker[activemq-artemis-broker]. É possível configurar a imagem e a versão usando a propriedade `quarkus.amqp.devservices.image-name`:"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/amqp-dev-services.adoc.po:112]:
```suggestion
msgstr "O Dev Services para AMQP usa a imagem  link:https://quay.io/repository/artemiscloud/activemq-artemis-broker[activemq-artemis-broker]. É possível configurar a imagem e a versão usando a propriedade `quarkus.amqp.devservices.image-name`:"
```

**Review by gastaldi — APPROVED** [review]
Muito bom! Parabéns

**Comment by gastaldi** [comment]
https://pr-preview-3-pt-quarkusio.surge.sh/guides/amqp-dev-services

**Comment by dellamas** [comment]
I will make some more corrections.

File: l10n/po/pt_BR/_guides/amqp-dev-services.adoc.po [hunk 2]
```diff
@@ -57,69 +55,61 @@ msgstr "o `amqp-host` ou `amqp-port` está configurado"
...
-msgstr "Por padrão, o Dev Services for AMQP escolhe uma porta aleatória e configura o aplicativo. É possível definir a porta configurando a propriedade `quarkus.amqp.devservices.port`."
+msgstr "Por padrão, o Dev Services para AMQP escolhe uma porta aleatória e configura a aplicação. É possível definir a porta configurando a propriedade `quarkus.amqp.devservices.port`."
 
 #. type: Title ==
 #: upstream/_guides/amqp-dev-services.adoc:46
-#, fuzzy, no-wrap
+#, no-wrap
 msgid "Configuring the image"
-msgstr "Configurar a imagem"
+msgstr "Configurarando a imagem"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/amqp-dev-services.adoc.po:107]:
```suggestion
msgstr "Configurando a imagem"
```

**Comment by gastaldi** [comment]
@dellamas looks like there are some missing translated parts in the doc? See https://pr-preview-3-pt-quarkusio.surge.sh/guides/amqp-dev-services#ativa%C3%A7%C3%A3odesativa%C3%A7%C3%A3o-do-dev-services-para-amqp and the end of the document

**Comment by gastaldi** [comment]
We can fix this in a separate PR. Merging this now

**Pull request merged**